server/app.js

```diff
@@ -31,21 +32,20 @@ require("./config/passport")(passport);
 // Routes
 app.use("/menu", menuRouter);
 app.use("/api/sessions", sessionRouter);
-
+app.use("/user", userRouter);
```

Comment:
i think we should follow /api/*s pattern here (i'll update the /menu one), can you update the /user routes? make sure the calls to the user API is also /api/users

Reply:
im gonna take out the users stuff we dont actually use them lol